src/__tests__/App.test.js

```diff
@@ -1,7 +1,24 @@
 import React from "react";
 import { shallow } from "enzyme";
 import App from "../App";
+import Home from "../components";
+import Login from "../components/login";
+import LoginView from "../views/login";
+import HomeView from "../views";
 
 it("renders without crashing", () => {
   shallow(<App />);
 });
+it("renders without crashing", () => {
+  shallow(<Home />);
+});
```

Comment:
Just testing the root component(App) wont crash is enough, no need to test all the other components here

Reply:
noted